contracts/validator/ValidatorSet.sol

```diff
@@ -114,10 +114,12 @@ contract ValidatorSet {
      *
      * @param _validator Validator address.
      * @param _beginHeight Begin height for the validator.
+     * @param _openKernelHeight Current open kernel height.
```

Comment:
this height should not be passed in, nor the new requirement be put.  Please revert this for `insertValidatorInternal` and `removeValidatorInternal`

Reply:
Reverted the change for `insertValidatorInternal` and `removeValidatorInternal`.